tests/foreman/maintain/test_health.py

```diff
@@ -27,6 +27,42 @@
 }
 
 
+@pytest.mark.satellite_iop_only
+def test_podman_login_check(request, sat_maintain):
+    """Test Podman login check with local Red Hat Lightspeed(IoP) Satellite.
+
+    :id: 70fd6d86-a647-442c-a971-cbd1207b734b
+
+    :setup: Configure a Satellite with local Red Hat Lightspeed(IoP) enabled.
+
+    :steps:
+        1. Run satellite-maintain update check.
+        2. Verify that the Podman login check passes.
+        3. Log out of the remote container registry.
+        4. Run satellite-maintain update check.
+        5. Verify that the Podman login check fails.
+
+    :Verifies: SAT-35282
+    """
+    iop_settings = settings.rh_cloud.iop_advisor_engine
+
+    request.addfinalizer(lambda: sat_maintain.podman_logout(iop_settings.registry))
+
+    check_description = 'Check whether podman needs to be logged in to the registry'
+    fail_message = '''You are using containers from registry.redhat.io,\n
+        but your system is not logged in to the registry, or the login expired.\n
+        Please login to registry.redhat.io.'''
```

Comment:
PRT seems to be failing due to incorrect fail_message string

```suggestion
    fail_message = '''You are using containers from registry.redhat.io,
    but your system is not logged in to the registry, or the login expired.
    Please login to registry.redhat.io.'''
```